docs/faq/proof-of-stake/pos-general-faq.md

```diff
@@ -0,0 +1,119 @@
+# **<i class="fa fa-ticket"></i> PoS General FAQ**
+
+---
+
+#### **1. What is Proof-of-Stake mining?**
+
+Besides Proof-of-Work (PoW) mining Decred has Proof-of-Stake (PoS) mining. In PoS mining the participants buy tickets using their Decred coins (DCR). These tickets go into the PoS pool and are randomly called upon to vote on a newly mined block. Every block 5 tickets are chosen from the pool and if at least 3 of those validate the mined block, the block is approved, added to the chain and both PoW miners and PoS miners are paid. If the block is not validated by the PoS miners the PoW miners do not get paid, but the PoS miners do. This is to incentivize PoW miners to mine according to the wishes of the PoS miners.
```

Comment:
I believe you are correct. They validate the last block so a new block can be mined/added to the chain.